astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+                 " " + observe_timezone)
+
+    # Output.
+    return ax
+
+
+def plot_parallactic(target, observer, time, ax=None, style_kwargs=None,
+                     dark_plot=False):
+    """
+    TODO:
+        1) dark_plot style
+        2) Parallactic angle equation? Move this to separate function?
+        3) observe_timezone -- update with info from observer?
+        4) Dealing with NaN or invalid values.
+
+    Makes a parallactic angle vs time plot.
+
+    If an ax object is passed in, plots an additional parallactic angle
+    plot on top.
+    Otherwise, creates a new ax object with a parallactic angle plot.
+
+    When a `Time` object with a single instance in time is used (e.g., if
+    len(time) = 1), a new `Time` object is created internally.  This new object
+    contains a sampling of times within a window centered on the input `Time`.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan.Observer`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        Used in generating data to plot.
+        Can be scalar (or len(time)=1) or contain multiple times.
+
+    ax : `~matplotlib.axes.Axes` or None, optional.
+        The axes this plot will be added to. If none are passed in, new axes
+        will be created.
+
+    style_kwargs : Dictionary or Empty, optional.
+        A dictionary of keywords passed into `matplotlib.pyplot.plot_date`
+        to set plotting styles.
+
+    dark_plot: Boolean.
+        Optional. False (default) or True.
+
+    Returns
+    -------
+    ax :  `~matplotlib.axes.Axes`
+        An axes object with added parallactic_angle vs. time plot.
+    """
+
+    # Set up plot axes and style if needed.
+    if ax is None:
+        ax = plt.gca()
+    if dark_plot is True:
+        raise NotImplementedError
+    if style_kwargs is None:
+        style_kwargs = {}
+    style_kwargs = dict(style_kwargs)
+    style_kwargs.setdefault('linestyle', '-')
+    style_kwargs.setdefault('color', 'b')
+    style_kwargs.setdefault('fmt', '-')
+
+    # If Time object is scalar, pad time window.
+    if time.isscalar:
+        time = time + np.linspace(-12, 12, 100)*u.hour
+
+    # Calculate parallactic angle.
+    declination = target.dec.to(u.rad)
+    latitude = observer.location.latitude.to(u.rad)
+    azimuth = observer.altaz(time, target).az.to(u.rad)
+
+    # Replace with Astroplan parallactic angle function call.
+    longitude = observer.location.longitude.to(u.rad).value
+    local_sidereal_time = time.sidereal_time('mean', longitude).to(u.rad).value
+    hour_angle = local_sidereal_time - target.ra.to(u.rad).value
+
+    numerator = np.sin(azimuth) * np.cos(latitude)
+    denominator = np.cos(declination) * (np.cos(azimuth)*np.cos(hour_angle) + np.sin(azimuth)*np.sin(hour_angle)*np.sin(latitude))
```

Comment:
This line is too long (I can't read it without scrolling left / right).
Please wrap it, putting the whole expression in parentheses, or use temp variables for sub-expressions (that's what I do, it's just as efficient in Python).